src/main/scala/dpla/ingestion3/harvesters/oai/DefaultSource.scala

```diff
@@ -31,17 +36,44 @@ class DefaultSource extends RelationProvider {
     new OaiRelation(endpoint, verb, metadataPrefix, harvestAllSets, setlist, blacklist)(sqlContext)
   }
 
+  def getMetadataPrefix(parameters: Map[String, String]): String = {
+    (parameters.get("metadataPrefix"), parameters.get("verb")) match {
+      case (Some(prefix), Some(verb)) => prefix
```

Comment:
I'm not sure we need this validation.  The `getVerb` validation you wrote before nicely checks for the presence of a supported verb.  In absence of a verb, the harvest will fail anyway.